packages/app/src/specs/AverageDuration.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div
+    v-if="props.gql.data?.__typename === 'CloudProjectSpec' && props.gql.data.averageDuration"
+    class="h-full grid text-gray-700 justify-end items-center"
+    data-cy="average-duration"
+  >
+    {{ getDurationString(props.gql.data.averageDuration) }}
+  </div>
+</template>
+
+<script setup lang="ts">
+
+import type { AverageDurationFragment } from '../generated/graphql'
+import { gql } from '@urql/vue'
+import { getDurationString } from '@packages/frontend-shared/src/utils/time'
+
+gql`
+fragment AverageDuration on RemoteFetchableCloudProjectSpecResult {
+  id
+  data {
+    ... on CloudProjectSpecNotFound {
+      retrievedAt
+    }
+    ... on CloudProjectSpec {
+      id
+      retrievedAt
+      averageDuration(fromBranch: $fromBranch)
```

Comment:
Is the `$fromBranch` variable value the same defined here: https://github.com/cypress-io/cypress/pull/21250/files#diff-2bc060c37de5ac22696888b84cc57c497d163d9a6506969f06bd65bf7e560b02R49? Or somewhere else?

Reply:
Correct.